store/sqlstore/job_store.go

```diff
@@ -42,16 +44,16 @@ func (jss SqlJobStore) Save(job *model.Job) (*model.Job, *model.AppError) {
 }
 
 func (jss SqlJobStore) UpdateOptimistically(job *model.Job, currentStatus string) (bool, *model.AppError) {
-	query := "UPDATE Jobs SET LastActivityAt = :LastActivityAt, Status = :Status, Progress = :Progress, Data = :Data WHERE Id = :Id AND Status = :OldStatus"
-	params := map[string]interface{}{
-		"Id":             job.Id,
-		"OldStatus":      currentStatus,
-		"LastActivityAt": model.GetMillis(),
-		"Status":         job.Status,
-		"Data":           job.DataToJson(),
-		"Progress":       job.Progress,
-	}
-	sqlResult, err := jss.GetMaster().Exec(query, params)
+	sql, args, err := jss.getQueryBuilder().
+		Update("").
```

Comment:
You can use here `Update("Jobs")` instead of `Udpate("").Table("Jobs")`

Reply:
Done.